### ISSUE TITLE
Report: ASSIN

### ISSUE THREAD

**Issue by arademaker** [fcbr/cl-wnbrowser#148]
http://nilc.icmc.usp.br/assin/

As the first step to evaluate if we can participate in this shared-task, it woud be nice to test the coverage of OWN-PT with these files.

**Comment by fcbr** [comment]
What's the strategy to handle the lack of lemmas in the data set?  Run it through Freeling first?

**Comment by arademaker** [comment]
Yes. It is what we have so far. Of course, it would be interesting to also identify the lack of words in the dict of FreeLing during the POS tagging, before the WSD. 

http://nlp.lsi.upc.edu/freeling/index.php?option=com_simpleboard&Itemid=65&func=view&id=3831&catid=3

http://nlp.lsi.upc.edu/freeling/doc/refman/classword.html#a3ede1c65b0329889d95f9146fde91236

Note that if we run Freeling with the updated version of OWN-PT, we can have a easy way to check a missing word in OWN-PT.

**Comment by vcvpaiva** [comment]
checking that the FreeLing dictionary has (or not) all these will be a good thing. and checking the POS will be even better. looking forward to the reports.

**Comment by fcbr** [comment]
We need #153 before this can be implemented.
